executable/detect.go

```diff
@@ -63,6 +66,7 @@ func (d Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error
 	}
 
 	if _, ok := m.Get("Main-Class"); ok {
+		d.Logger.Info("PASSED: 'Main-Class' manifest attribute found")
```

Comment:
well, it's not an error in that case, but in the case the user uses verbose then I guess it could help